Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs

```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
             var firstLetter = (int)rangeParts[0][0];
             var secondLetter = (int)rangeParts[1][0];
 
-            if (secondLetter < firstLetter)
+            /* if (secondLetter < firstLetter)
+             {
+                 throw new Exception("Invalid range specified.");
+             }*/
+
+
+            // Handle also columns after AA
+            Regex regexRange = new Regex(@"([A-Z]+)\d+");
```

Comment:
We could initialize the regex as a static member and compile it to reuse it between calls